### PR TITLE
Add function to convert interval day to second to millis

### PR DESCRIPTION
**Pull request by raghavsethi**

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java [hunk 1]
```diff
@@ -951,6 +955,16 @@ public void testParseDuration()
         assertInvalidFunction("parse_duration('abc')", "duration is not a valid data duration string: abc");
     }
 
+    @Test
+    public void testToMilliseconds()
+            throws Exception
+    {
+        assertFunction("to_milliseconds(parse_duration('1ms'))", BigintType.BIGINT, 1L);
```
Comment (nezihyigitbasi) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java]: you can add a test for `ns`.

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java [hunk 2]
```diff
@@ -951,6 +955,16 @@ public void testParseDuration()
         assertInvalidFunction("parse_duration('abc')", "duration is not a valid data duration string: abc");
     }
 
+    @Test
+    public void testToMilliseconds()
```
Comment (nezihyigitbasi) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java]: you can be more specific with the method name to include that you are converting `INTERVAL_DAY_TO_SECOND`

Comment (raghavsethi) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java]: Good point.

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java (hunk 1 above)
Comment (raghavsethi) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java]: Done.

**Pull request closed without merging**